extensions/sonarqube/overview.md

```diff
@@ -1,50 +1,55 @@
 
 ![Branches](img/branches.png)
 
-If you configure your build definition as a build validation for pull requests of that project (this can be done on "Branch policies"), SonarQube will also analyze the code changes and decorate the pull request with comments and overall status so that you can merge with confidence:
+If you configure your build definition as a build validation for pull requests of that project (this can be done in 'Branch Policies'), SonarQube will also analyze the code changes and decorate the pull request with comments and overall status so that you can merge with confidence.
+
+When a build is run on a branch, the extension automatically configures the analysis to be pushed to the relevant project branch on SonarQube.
 
 ![PR-Decoration](img/pull-request-decoration.png)
 
 **Important note**: to activate pull request decoration, you must specify a user token in the "General Settings > Pull Requests" administration page of your project in SonarQube.
 
-### Quality Gate Status
-The **Publish Quality Gate Result** task waits for the analysis report to be consumed by the SonarQube in order to flag the build job with the Quality Gate status. The Quality Gate is a major, out-of-the-box, feature of SonarQube. It provides the ability to know at each analysis whether an application passes or fails the release criteria. In other words it tells you at every analysis whether an application is ready for production "quality-wise".
+## About the SonarQube Azure DevOps Marketplace Extension
+This extension provides the following features:
+* A dedicated **SonarQube EndPoint** that defines the SonarQube server to be used.
+* Branch and **Pull Request** analysis in your projects.
+* Three build tasks to get your projects analyzed easily:
+  * **Prepare Analysis Configuration:** Configures all the required settings prior to executing a build. This task is mandatory. For .NET solutions or Java projects, this task helps SonarQube seamlessly integrate with MSBuild, Maven and Gradle tasks.
+  * **Run Code Analysis:** Executes the source code analysis. This task isn't required for Maven or Gradle projects.
+  * **Publish Quality Gate Result:** Displays the Quality Gate status in the build summary. This tasks is optional, as it may increase the overall build time.
 
-Example of a passing Quality Gate:
-![Passed Qualiy Gate](img/sq-analysis-report-passed.png)
+**Note for TFS installations older than TFS 2017 Update 2**: to install the extension, please follow instructions
+available on the ["SonarQube Extension 3.0" documentation page](https://docs.sonarqube.org/display/SCAN/SonarQube+Extension+3.0).
 
-Example of a failing Quality Gate:
-![Failed Qualiy Gate](img/sq-analysis-report-failed.png)
+## Highlights
```

Comment:
I'd suggest merging this "highlights" section with the overall Benefits suggestion above. It feels off to have some things covered in one section and then more down here. In fact it might be nice to include screenshots of the build tasks added to a build pipeline (I can show you where/what I mean). 

In fact I wonder if this entire "About" section should be merged with the content above, with screenshots added as appropriate. I can help you order/prioritize it if you're not sure how to blend.